helm/values-dev.yaml

```diff
@@ -37,6 +37,6 @@ pdb:
 configEnvSpecific:
   INTERNAL_API_SERVICE_URL: "https://api.dev.selfcare.pagopa.it"
   MS_EXTERNAL_INTERCEPTOR_LOG_LEVEL: "DEBUG"
-  SAP_ALLOWED_INSTITUTION_TYPES: "PA,GSP,SA,SA,SCP"
+  SAP_ALLOWED_INSTITUTION_TYPES: "PA,GSP,SA,AS,SCP"
```

Comment:
If there is no difference among the three env, maybe we could move this variable into values.yaml